Ladybird/Qt/main.cpp

```diff
@@ -54,9 +55,40 @@ static ErrorOr<void> handle_attached_debugger()
     return {};
 }
 
+class LadybirdApplication : public QApplication {
+public:
+    LadybirdApplication(int& argc, char** argv)
```

Comment:
Does this need the Q_OBJECT() macro? If so, it'll need its own .h/cpp for the gn build's logic to be happy

Reply:
Otherwise, this looks good :+1:

Reply:
I'm not sure, but this came directly from Qt's macOS example:
https://doc.qt.io/qt-5/qfileopenevent.html#macos-example

Reply:
> The Q_OBJECT macro must appear in the private section of a class definition that declares its own signals and slots or that uses other services provided by Qt's meta-object system.

Since we're not using any of that here, and everything builds / runs fine without it, I think it's not needed 🤷